src/web/Fig.Web/Pages/Groups.razor.css

```
@@ -0,0 +1,397 @@
+    border-radius: 4px;
+    border: 1px solid rgba(255, 255, 255, 0.15);
+    cursor: pointer;
+    transition: transform 0.15s ease;
+}
+
+.color-preview:hover {
+    transform: scale(1.1);
+}
+
+/* ===== ListBox overrides (matches Settings page) ===== */
+::deep .groups-listbox {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)